Library/Formula/unarj.rb

```diff
@@ -0,0 +1,15 @@
+class Unarj < Formula
+  homepage "http://www.arjsoftware.com/files.htm"
+  url "http://pkgs.fedoraproject.org/repo/pkgs/unarj/unarj-2.65.tar.gz/c6fe45db1741f97155c7def322aa74aa/unarj-2.65.tar.gz"
+  sha1 "e30242ab0cb521bae89472a552219a06fdddb389"
+  version "2.65"
```

Comment:
Yep - the version can be detected from the filename, so the formula already knows what the version is.